### PR TITLE
Fix part of #5195: RestrictedApi. Using a private API

### PR DESCRIPTION
**Pull request by sichchurov**
## Explanation
Fix part of #5195: I have modified method setButtonTint to avoid an error due to using a restricted API.
This method(bindingAdapter) is not set in any of the layouts. Only test is written for it.
So it is safety to modify the test also. If not modify a test - it is failed then.

**Lint report before**
callingNewApiBefore

**Lint report after**
callingNewApiBefore

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- For PRs introducing new UI elements or color changes, both light and dark mode screenshots must be included
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by sichchurov** [comment]
@adhiamboperes, PTAL
Thanks

File: app/src/main/java/org/oppia/android/app/databinding/AppCompatCheckBoxBindingAdapters.java [hunk 1]
```diff
@@ -4,15 +4,18 @@
...
 import androidx.databinding.BindingAdapter;
 
 /**
  * Custom data-binding adapters for {@link AppCompatCheckBox}s.
  */
 public final class AppCompatCheckBoxBindingAdapters {
-  /** Sets the button tint for the specified checkbox, via data-binding. */
+  /**
+   * Sets the button tint for the specified checkbox, via data-binding.
+   */
```
Comment (adhiamboperes) [inline app/src/main/java/org/oppia/android/app/databinding/AppCompatCheckBoxBindingAdapters.java]: Single line KDocs need to be on one line, as it was before. Please revert this. You can check out the kotlin style guide that is linked in our wiki.

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @sichchurov.

The solution looks good, but it is not correct to say that this attribute is unused. I searched, and found multiple usages. See screenshot:
Screenshot 2023-10-20 at 15 56 48

File: app/src/main/java/org/oppia/android/app/databinding/AppCompatCheckBoxBindingAdapters.java (hunk 1 above)
Comment (sichchurov) [reply app/src/main/java/org/oppia/android/app/databinding/AppCompatCheckBoxBindingAdapters.java]: My mistake. I have been looking using double shift. But forgot about ctrl +shift + f.
About code style - I'll read read more carefully this section

**Comment by sichchurov** [comment]
@adhiamboperes, PTAL
Thanks

**Review by adhiamboperes — APPROVED** [review]
LGTM, thanks @sichchurov!

**Pull request merged**